CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
+## [2.1.15] - 2021-11-10
+### Fixed
+- update Elastigroup Aws `get_deployment_status` call
```

Comment:
change to
` Change the api call made by elastigroup aws 'get_deployment_status'`
what do you think?